packages/api/internal/cache/instance/instance.go

```diff
@@ -50,8 +51,14 @@ func NewInstanceInfo(
 	EnvdAccessToken *string,
 	BaseTemplateID string,
 ) *InstanceInfo {
+	// TODO: Temporary workaround, until all orchestrators report this
```

Comment:
I could also separate the PR and add the fallback only to one of the services (api/orchestrator). This is caused by the circular dependency